### ISSUE TITLE
CFD Offer screen price values are somehow not updated

### ISSUE THREAD

**Issue by da-kami** [itchysats/10101#315]

**Comment by da-kami** [comment]
I think the problem is that we don't handle errors gracefully, so we stop actually syncing if e.g. the maker is gone. 

I am adding `try {} catch` blocks around the API calls, let's see if that does it

**Comment by da-kami** [comment]
OK, what I did is probably not bad, but it's not the problem. We run into the error when we receive a `None` value it appears, i.e. when the `offer` is not set. Fixing this with: https://github.com/itchysats/10101/pull/319

Once thing I noticed is, that the maker actually does not receive any price updates - at least we don't log any. We see an initial price quote from bitmex upon startup, but that's it. So, either the price is actually not changing or there is a problem in the maker.

**Comment by da-kami** [comment]
Note: We actually *do* handle expections when fetching from rust gracefully, we do this in `runPeriodically` :)

**Comment by klochowicz** [comment]
it seems to be erroring deep down in `bitmex_stream`, you can see it with `RUST_LOG=trace`

```
2022-11-23 04:34:26 TRACE maker::bitmex: Received message from bitmex text={"table":"instrument","action":"update","data":[{"symbol":"XBTUSD","fairPrice":16491.56,"fairBasis":1.53,"markPrice":16491.56,"openValue":352514993592,"timestamp":"2022-11-23T04:34:25.000Z","indicativeSettlePrice":16490.03}]}
2022-11-23 04:34:26 TRACE maker::bitmex: Not a 'table' message, skipping... text={"table":"instrument","action":"update","data":[{"symbol":"XBTUSD","fairPrice":16491.56,"fairBasis":1.53,"markPrice":16491.56,"openValue":352514993592,"timestamp":"2022-11-23T04:34:25.000Z","indicativeSettlePrice":16490.03}]} e=missing field `bidPrice` at line 1 column 223
```

after the initial one, we don't put anything on the stream - we ignore the messages because they're not as we expect them to be.

**Comment by da-kami** [comment]
Fixed with #324
